apps/dsp-app/src/app/workspace/resource/representation/document/document.component.html

```diff
@@ -10,7 +10,7 @@
 
     <pdf-viewer
         class="pdf-viewer"
-        [src]= "{ url: src.fileValue.fileUrl, withCredentials: true }"
+        [src]= "{ url: src.fileValue.fileUrl,  withCredentials: true }"
```

Comment:
does `withCredentials` have a fear of commas? :)

Reply:
I just needed some space ;-)